cmd/titus-imds-cni/env.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	"github.com/Netflix/titus-executor/utils/k8s"
 
-	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/proto" // nolint: staticcheck
```

Comment:
As a heads up: these were due to the new version of `golangci-lint` not being happy that this package is marked as deprecated.  It's actually a major version update, which I didn't want to take on as part of this change.  For more info, see here: https://blog.golang.org/protobuf-apiv2